pkg/api/controller.go

```diff
@@ -3006,6 +3012,65 @@ func (c *Controller) GetCurrentUser(w http.ResponseWriter, r *http.Request) {
 	writeResponse(w, http.StatusOK, response)
 }
 
+func (c *Controller) sendResetPasswordEmail(email []string, token string) error {
+	err := c.Emailer.SendEmail(email, token, token, []string{})
+	if err != nil {
+		c.Logger.Error("Error sending email")
```

Comment:
This log is not useful.  How do you debug an error with just a log line that says "Error sending email"?  And if there are 10 organizations with 100 users each?

Please add (all) emails, the error, and more details as fields on the log.

Reply:
Add logging with fields. It should be noted, that besides for the email address there are no other fields with relevant info (i.e. org id is not available in OSS)